### PR TITLE
Fix a few possible issues with the catalog as seen in Firebase

### PR DESCRIPTION
**Pull request by twaddington**
**Why are we doing this? (w/ JIRA link if applicable)**
- https://console.firebase.google.com/project/simplye-nypl/crashlytics/app/android:org.nypl.simplified.simplye/issues/e8fea58b44d1fa66eb10da787ddc2bbe
- https://console.firebase.google.com/project/simplye-nypl/crashlytics/app/android:org.nypl.simplified.simplye/issues/228ac27834b3139445bf35d4134f4698
- https://console.firebase.google.com/project/simplye-nypl/crashlytics/app/android:org.nypl.simplified.simplye/issues/039396d4fac6c99ad031d1ba38124dca

**How should this be tested? / Do these changes have associated tests?**
n/a

**Dependencies for merging? Releasing to production?**
n/a

**Has the application documentation been updated for these changes?**
n/a

**Did someone actually run this code to verify it works?**
@twaddington ran the vanilla app.

### CODE REVIEW

**Comment by io7m** [comment]
I'd _really_ like to know how we managed https://console.firebase.google.com/project/simplye-nypl/crashlytics/app/android:org.nypl.simplified.simplye/issues/228ac27834b3139445bf35d4134f4698?time=last-seven-days&sessionId=5F324AE8003F00016C5D7A5FE769989D_DNE_0_v2

File: simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFeedViewModel.kt [hunk 1]
```diff
@@ -413,8 +414,13 @@ class CatalogFeedViewModel(
           title = title
         )
 
-      is CatalogFeedArgumentsLocalBooks ->
-        throw IllegalStateException("Cannot transition from a local feed to a remote feed.")
+      is CatalogFeedArgumentsLocalBooks -> {
+        val oldTitle = this.feedArguments.title
+        val newTitle = title
+        throw IllegalStateException(String.format(
+          Locale.US, "Can't transition local to remote feed: {} -> {}", oldTitle, newTitle
```
Comment (io7m) [inline simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFeedViewModel.kt:421]: Wrong kind of format string. You want "%s -> %s".

Comment (twaddington) [reply simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFeedViewModel.kt:421]: Ah, damn. Good catch!

**Comment by twaddington** [comment]
> Format string issue.

Fixed!

**Pull request merged**